Formula/lgogdownloader.rb

```diff
@@ -0,0 +1,37 @@
+class Lgogdownloader < Formula
+  desc "Open source downloader for GOG.com games"
+  homepage "https://sites.google.com/site/gogdownloader/"
+  url "https://sites.google.com/site/gogdownloader/lgogdownloader-3.2.tar.gz"
+  sha256 "9e60af50c18533884ca317778b291152d5f04d93dfd3dc2579f6f965538b2031"
+
+  depends_on "cmake" => :build
+  depends_on "help2man" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "boost"
+  depends_on "htmlcxx"
+  depends_on "jsoncpp"
+  depends_on "liboauth"
+  depends_on "rhash"
+  depends_on "tinyxml2"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    require "pty"
+
+    ENV["XDG_CONFIG_HOME"] = testpath
```

Comment:
I have a stupid question. What happens if we don't set this?

Reply:
Not stupid at all! See [the relevant discussion](https://github.com/Homebrew/homebrew-core/pull/19189#discussion_r144208451).

tl;dr If we don’t set this, the CI will run fine but `brew test` will inexplicably fail for most end users.